README.md

```diff
@@ -85,6 +85,12 @@ Integration tests can be run on any deployed DC/OS cluster. For installation ins
 
 Integration tests are installed via the [dcos-integration-test](./packages/dcos-integration-test/) Pkgpanda package.
 
+Integration test files are stored on the DC/OS master node at `/opt/mesosphere/active/dcos-integration-test`.
+Therefore, in order to test changes to test files, move files from `packages/dcos-integration-test/extra/` in your checkout to `/opt/mesosphere/active/dcos-integration-test` on the master node.
+
+The canonical source of the test suite's results is the continuous integration system.
```

Comment:
FWIW I plan to take a brief look with @mellenburg to see if we can at least roughly scope the problems, probably in order to create a follow-up JIRA.